transcom/mymove
# B 20066 int

## [B-20066](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20066)

## Summary
This ended up being a very simple two-line change to add the Space Force branch to the options used for PDF Generation, which is used in QAE/CSR and PPM PDF generation.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a new Milmove Space Force Service Member
2. Create an HHG shipment, complete and submit it.
3. Log in as a QAE office user, locate the shipment.
4. Create a shipment report from the QAE reports tab.
5. Create a counseling report from the QAE reports tab.
6. Download both reports. Ensure that the Space Force branch is displayed like in screenshot 1 below in both.
7. Return to the Service Member. 
8. Create a new shipment - a PPM with an advance request, submit it.
9. Log in as a service counselor, approve the PPM and advance. Download the AOA, check the branch is displayed like in screenshot 2 below.
10. Log back in as a service member, complete the customer closeout process for the PPM.
11. Log back in as a service counselor, approve the PPM documents to complete the closeout process.
12. Download the payment packet, check the branch is displayed like in screenshot 3 below.

> [!NOTE]  
> Please ensure the database migrations run properly. There's been some weirdness between INT and Main when it comes to the migration files.

## Screenshots
1. QAE/CSR Report with branch added
<img width="815" alt="image" src="https://github.com/user-attachments/assets/1e668382-0fa0-4bef-a60e-4c45f9b3ec23">

2. AOA Packet with branch added
<img width="566" alt="image" src="https://github.com/user-attachments/assets/6b15d5b4-60ad-4709-81d2-9285e2fa0c69">

3. Payment Packet with branch added
<img width="566" alt="image" src="https://github.com/user-attachments/assets/c994b358-8ac8-4e93-961a-e5fd18db5ab5">
